CUDADataFormats/Common/interface/HostProduct.h

```diff
@@ -12,18 +13,18 @@ class HostProduct {
   HostProduct(HostProduct&&) = default;
   HostProduct& operator=(HostProduct&&) = default;
 
-  explicit HostProduct(cms::cuda::host::unique_ptr<T>&& p) : hm_ptr(std::move(p)) {}
+  //   explicit HostProduct(cms::cuda::host::unique_ptr<T>&& p) : hm_ptr(std::move(p)) {}
   explicit HostProduct(std::unique_ptr<T>&& p) : std_ptr(std::move(p)) {}
 
-  auto const* get() const { return hm_ptr ? hm_ptr.get() : std_ptr.get(); }
+  auto const* get() const { return std_ptr.get(); }  // hm_ptr ? hm_ptr.get() : std_ptr.get(); }
```

Comment:
given that the test pass, I assume this means that the previous implementation can be removed here too.